hilti/src/ast/scope-lookup.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+
+#include <utility>
+
+#include <hilti/ast/ctors/enum.h>
+#include <hilti/ast/declarations/constant.h>
+#include <hilti/ast/declarations/module.h>
+#include <hilti/ast/scope-lookup.h>
```

Comment:
Let's include the header matching this impl file first,
```.cpp
#include "hilti/ast/scope-lookup.h"
```

This is used by some tools to e.g., infer which header should declare entities defined in some impl file. This is also the a widely used style, see e.g., https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes (which I do not agree with in many places, but here it IMHO makes good sense).

Reply:
I'd be fine switching to that style, but not sure I like doing that incrementally just for a individual files, that would just make things inconsistent. One wouldn't even know if for a specific file, header inference works or not. So staying with current style until we switch over.